project/make/dynbinary_gccgo

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Let's rename this to `dyngccgo` for now so it matches `binary`->`dynbinary` (`gccgo`->`dyngccgo`), and then revisit the naming scheme later.